### ISSUE TITLE
net6.0-windows10.0.18362.0 and windows 7

### ISSUE THREAD

**Issue by AlexShovkoplyas** [Kinnara/ModernWpf#466]
Please suggest if using <TargetFramework>net6.0-windows10.0.18362.0</TargetFramework> would work on Windows 7 OS? There is no possibility to test it.

**Comment by nabeelio** [comment]
Yeah, I'm wondering if there's a `net6.0-windows7` target? I need to look to see if that TFM exists

**Comment by Kinnara** [comment]
Closing as not planned during the 1.x backlog review. This request targets legacy 0.9-era behavior, an excluded integration/platform, or functionality outside the current WinUI/WPF Fluent parity scope. If there is a current 1.x regression with a minimal reproduction, please open a focused issue against the supported targets.
